core-concepts/datanft-and-datatoken.md

```diff
@@ -43,6 +43,16 @@ Here's an example.
 * In step 1, Alice **publishes** her dataset with Ocean: this means deploying an ERC721 data NFT contract (claiming copyright/base IP), then an ERC20 datatoken contract (license against base IP).
 * In step 2, she **mints** some ERC20 datatokens and **transfers** 1.0 of them to Bob's wallet; now he has a license to be able to download that dataset.
 
+### TemplateIds
+
+Each data NFT or a datatoken is cloned from pre-defined template contracts. The `templateId` parameter refers to the template from which a data NFT or datatoken is created. The `templateId` can be set while creating data NFT/datatoken. Currently, Ocean protocol supports 1 template type for dataNFT and 2 template variants for datatokens, namely: **regular template** and **enterprise template**. Each template supports the same interface but differs in the underlying implementation and can have additional features.
```

Comment:
```suggestion
Each data NFT or a datatoken is minted from pre-defined template contracts. The `templateId` parameter refers to the template from which a data NFT or datatoken is created. The `templateId` can be set while creating data NFT/datatoken. Currently, Ocean protocol supports 1 template type for data NFT and 2 template variants for datatokens, namely: **regular template** and **enterprise template**. Each template supports the same interface but differs in the underlying implementation and can have additional features.
```

Reply:
cloned sounds better, because we do actually clone the contract when deploying a new one